zwave_js_server/model/value.py

```diff
@@ -63,9 +63,9 @@ def readable(self) -> Optional[bool]:
         return self.data.get("readable")
 
     @property
-    def writable(self) -> Optional[bool]:
+    def writeable(self) -> Optional[bool]:
         """Return writeable."""
-        return self.data.get("writable")
+        return self.data.get("writeable")
```

Comment:
Have we checked this spelling against real data? The docs are inconsistent about what spelling is the correct one.

https://zwave-js.github.io/node-zwave-js/#/api/valueid?id=valuemetadata

Reply:
Yep, I've checked our fixtures:
- https://github.com/home-assistant-libs/zwave-js-server-python/blob/master/test/fixtures/multisensor_6_state.json#L77
- https://github.com/home-assistant-libs/zwave-js-server-python/blob/master/test/fixtures/basic_dump.txt#L1
and the source code of zwave-js:
- https://github.com/zwave-js/node-zwave-js/blob/0a7bdb15dd50ecc5aa146c12c20b360320b9e169/packages/core/src/values/Metadata.ts#L43

Reply:
Great!